discodeit/src/main/java/com/sprint/mission/discodeit/exception/GlobalExceptionHandler.java

```diff
@@ -1,36 +1,75 @@
 package com.sprint.mission.discodeit.exception;
 
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 
-import java.util.NoSuchElementException;
-
+@Slf4j
 @RestControllerAdvice
 public class GlobalExceptionHandler {
 
-  @ExceptionHandler(IllegalArgumentException.class)
-  public ResponseEntity<String> handleException(IllegalArgumentException e) {
-    e.printStackTrace();
-    return ResponseEntity
-        .status(HttpStatus.BAD_REQUEST)
-        .body(e.getMessage());
+  // 커스텀 에러 처리
+  @ExceptionHandler(DiscodeitException.class)
+  public ResponseEntity<ErrorResponse> handleCustomException(DiscodeitException e) {
+    log.error("커스텀 예외 발생 : code={}, message={}", e.getErrorCode(), e.getMessage());
```

Comment:
```suggestion
  log.error("커스텀 예외 발생 - code: {}, message: {}",   e.getErrorCode(), e.getMessage(), e);
```

이처럼 exception 스택 트레이스도 로그에 포함시켜주세요!